### PR TITLE
add custom useGetArticle hook

### PR DESCRIPTION
**Pull request by webdevelukas**

### CODE REVIEW

File: src/hooks/useGetArticle.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { getArticle } from "../api/fetch";
+
+export default function useGetArticle({ match }) {
```
Comment (lmachens) [inline src/hooks/useGetArticle.js]: This hook has a dependency to ReactRouters `match`  and a form now. 
Better isolate it and add the `articleId` and the updated `article` (data) as arguments. 
With this change, you can use this hook in more scenarios like "I like to load an old state of an article"

File: src/hooks/useGetArticle.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      headers: {
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(data)
+    });
+  }
+
+  useEffect(() => {
+    getArticle(articleId).then(fetchedArticle => setArticle(fetchedArticle));
+    // eslint-disable-next-line
```
Comment (lmachens) [inline src/hooks/useGetArticle.js]: This shouldn't be necessary if you update your eslint rules

Comment (webdevelukas) [reply src/hooks/useGetArticle.js]: I haven't seen an easy way to turn it off. I also read, that it's inadvisable to do it. Didn't saved the source. :/

File: src/hooks/useGetArticle.js (hunk 1 above)
Comment (webdevelukas) [reply src/hooks/useGetArticle.js]: Thanks for the feedback, i will try to do it. :)

Comment (webdevelukas) [reply src/hooks/useGetArticle.js]: Don't know if i did it right now, but i tried to split the code a bit more. I have in mind that we're already talking about loading an old state, but do you have an example for this? Cause now i didn't send `article` (data) as argument to the `useGetArticle` hook.

**Pull request merged**